src/tools/genpolicy/src/policy.rs

```diff
@@ -327,6 +327,36 @@ pub struct ExecProcessRequestDefaults {
     regex: Vec<String>,
 }
 
+/// IP address struct schema for genpolicy-settings.json.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct IPAddress {
+    /// IP family; 0 for ipv4 or 1 for ipv6
+    family: u8,
+
+    /// IP address
+    address: String,
+
+    /// IP mask
+    mask: String,
```

Comment:
Where in the codebase can I find out if this is actually a string? Shouldn't it be u8 or something?

Reply:
This info looks like is in an autogenerated file ignored by git. Looks like it is a string

<img width="310" alt="image" src="https://github.com/user-attachments/assets/d3a0f71e-3475-49cc-a56f-9ec1390b9d99" />